README.md

```diff
@@ -1,63 +1,12 @@
 # MiniPlaner
 
-Software zum effizienten Erstellen hochwertiger Messdienerpläne.
+An application to efficiently create high-quality service plans.
 
-<img src="minis64.png" alt="Logo"/>
+## Dependencies
 
-## Beschreibung
+* The Meson Build System: [Getting meson](https://mesonbuild.com/Getting-meson.html)
+* wxWidgets (discoverable via a config tool like `wx-config`): [Downloads - wxWidgets](https://www.wxwidgets.org/downloads/)
```

Comment:
add minimum version? 3.0.4 should be fine.

Reply:
I think it would make more sense to enforce a minimum version via meson and not specify the version in more places, as it would be hard to keep up with those.

What do you think?

Reply:
agreed

Reply:
Done.